Dockerfile

```diff
@@ -92,7 +117,11 @@ COPY --from=pip /usr/local/lib/python2.7/site-packages/psutil /usr/local/lib/pyt
 COPY --from=bess-build /opt/bess /opt/bess
 COPY --from=bess-build /bin/bessd /bin
 COPY conf /opt/bess/bessctl/conf
-RUN ln -s /opt/bess/bessctl/bessctl /bin
+RUN ln -s /opt/bess/bessctl/bessctl /bin; \
+  if [ x"$ENABLE_NTF" != "x" ]; then \
+    sed -i 's/"enable_ntf": false/"enable_ntf": true/g' \
+    /opt/bess/bessctl/conf/spgwu.json; \
+  fi
```

Comment:
Delete this change. Add "comment" in the json file instead. This is customized by user per instance. They can uncomment. example -
```
"": "Enable network tokens on uplink and downlink",
"": "enable_ntf: true", 
```

https://github.com/omec-project/upf-epc/blob/4e16c8ed5b8484e8db551ab26f40c328683ebb9c/conf/spgwu.json#L2-L4

Reply:
i see that you already have the json comment.

Reply:
Okay good to know - the block in the Dockerfile was to enable it by default because I was unsure how spgwu.conf is distributed.  It makes sense to do it there.

Stylistically How should this look? Like a default value:

    "enabled": "false"

or an unset value:

    "": "enabled: false"

Reply:
"": "enabled: false"